vocabularies/UI.xml

```diff
@@ -1004,6 +1004,7 @@ The trend is
               <String>UI.KPI</String>
               <String>UI.LineItem</String>
               <String>UI.MediaResource</String>
+              <String>UI.Note</String>
```

Comment:
I guess the purpose of the change is similar to `UI.LineItem` in CDS it's shown `#LINEITEM_REFERENCE`. With the addition of UI.Note, would the following annotation appear in CDS? 
`````
@UI.facet: [
{
      id:            'NotesSection',
      purpose:       #STANDARD,
      type:          #NOTE_REFERENCE,
      label:         'Note Section',
      position:      10,
      targetElement: '_NoteBasic'
  } ]
````
It seems that the singular is almost always used here. In the case of the Notes, it is usually a list. So shouldn't be called #NOTES or NOTELIST?

Reply:
@thomas-kaltenbach Is this comment still relevant? If not, please resolve this conversation.

Reply:
It's not relevant anymore